apps/pie-storybook/stories/pie-switch.stories.ts

```diff
@@ -119,6 +119,10 @@ const Template : TemplateFunction<SwitchProps> = (props) => {
         required,
     } = props;
 
+    // can put the label adjacent to the switch OR put directly in the switch component(check the switch index.ts file to see example)
+    // <pie-form-label for="pie-switch">
+    //     label slot content
+    // </pie-form-label>
```

Comment:
This is the approach I would suggest we go with for usage. It is more flexible as the alternative means passing a selector down to use with the label